tests/table_test.go

```diff
@@ -394,14 +394,14 @@ func TestTable(t *testing.T) {
 				changerBinaryPath,
 			},
 			verify: func(t *testing.T, vcMock *vcmock.VersionController, runData runData) {
-				require.Len(t, vcMock.PullRequests, 1)
+				require.Len(t, vcMock.PullRequests, 2)
 				assert.Contains(t, runData.logOut, "Running on 2 repositories")
 				assert.Contains(t, runData.logOut, "Cloning and running script")
 
 				assert.Equal(t, `The new branch already exists:
   owner/already-existing-branch
 Repositories with a successful run:
-  owner/should-change #1
+  owner/should-change #2
 `, runData.out)
```

Comment:
I would expect any created pull request to be shown here, even if the branch already exists. (And maybe even the ones that exist before the run?) Let us expand the test to include a third repo, where both the branch and pull request already exist.

I would expect the log final log output to be either:

```
The new branch already exists, but a pull request was created:
  owner/already-existing-branch #2
The new branch already exists
  owner/already-existing-branch-and-pr
Repositories with a successful run:
  owner/should-change #3
```
**or**
```
The new branch already exists
  owner/already-existing-branch-and-pr
Repositories with a successful run:
  owner/already-existing-branch #2
  owner/should-change #3
```
**or**
```
The new branch already exists, but a pull request was created:
  owner/already-existing-branch #2
The new branch already exists
  owner/already-existing-branch-and-pr #1
Repositories with a successful run:
  owner/should-change #3
```
**or**
```
The new branch already exists
  owner/already-existing-branch-and-pr #1
  owner/already-existing-branch #2
Repositories with a successful run:
  owner/should-change #3
```

Thoughts?

Reply:
Yes absolutely agree, last one.

I changed it a bit so it now also links existing pull requests as well as it links newly created ones for existing branches.